library/patient.inc

```diff
@@ -137,6 +138,10 @@ function getInsuranceProvidersExtra() {
            "," . $row['zip'] . ")";
         break;
         case $GLOBALS['insurance_information'] = '5':
+	       $returnval[$row['id']] = $row['name'] . " (" . $row['line1'] . "," . $row['city'] .
+               "," . $row['state'] . "," . $row['zip'] . ")";
```

Comment:
No space between comma and next word?

This is returning `Name (Line1,City,ST,ZIP)` which looks cramped, suggest `Name (Line1, City, ST ZIP)